skills/csharp/experimental/itsmskill/Dialogs/Teams/TicketTaskModule/UpdateTicketTeamsImplementation.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace ITSMSkill.Dialogs.Teams.TicketTaskModule
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using ITSMSkill.Extensions.Teams;
+    using ITSMSkill.Extensions.Teams.TaskModule;
+    using ITSMSkill.Models;
+    using ITSMSkill.Services;
+    using ITSMSkill.TeamsChannels;
+    using ITSMSkill.TeamsChannels.Invoke;
+    using Microsoft.Bot.Builder;
+    using Microsoft.Bot.Schema.Teams;
+    using Microsoft.Extensions.DependencyInjection;
+
+    /// <summary>
+    /// UpdateTicket Handler for Updating Ticket OnFetch and OnSumbit Activity for TaskModules
+    /// </summary>
+    [TeamsInvoke(FlowType = nameof(TeamsFlowType.UpdateTicket_Form))]
```

Comment:
same in this class

Reply:
removed the classes as i implemented them in another PR.